src/fides/api/api/v1/endpoints/dataset_endpoints.py

```diff
@@ -573,3 +575,47 @@ def delete_dataset(
         "Deleting dataset '{}' for connection '{}'", fides_key, connection_config.key
     )
     dataset_config.delete(db)
+
+
+@router.get(
+    f"/filter{DATASETS}",
+    dependencies=[Security(verify_oauth_client, scopes=[DATASET_READ])],
+    response_model=List[Dataset],
+)
+def get_ctl_datasets(
+    db: Session = Depends(deps.get_db),
+    remove_saas_datasets: bool = True,
+    only_unlinked_datasets: bool = False,
+) -> List[Dataset]:
+    """
+    Returns all CTL datasets .
+    """
+
+    logger.info(
+        f"Finding all datasets {remove_saas_datasets=} {only_unlinked_datasets=}"
+    )
+    filters = []
+    if only_unlinked_datasets:
+        unlinked_subquery = select([DatasetConfig.ctl_dataset_id])
```

Comment:
How does this determine that the dataset is unlinked?

Reply:
Good question. This subquery gets a list of all the datasets that have an id saved on a `DatasetConfig`. It's return as a list of ids. Any dataset that has an id on one of those records is "linked".

The filter on line 600 is saying return all the datasets that don't have their id in the list of linked ids